### ISSUE TITLE
Group all templates with correct naming of bundles/extensions 

### ISSUE THREAD

**Issue by agoat** [contao/core-bundle#1222]
When creating a new template, extensions (bundles) are not grouped correctly if the name (path) does  not end with `*-bundle`. They are all grouped under the vendor name only.

The pattern in `preg_replace` searches only for a specific extension:
https://github.com/contao/core-bundle/blob/133cebc0a9716f935637dcb64cd46a5b37920d02/src/Resources/contao/dca/tl_templates.php#L247-L252

We can catch all bundles/extension by changing the pattern to:
```php
$strModule = preg_replace('@^(vendor/(.*?/.*?)/|system/modules/(.*?)/).*$@', '${2}${3}', strtr($strRelpath, '\\', '/'));
```

**Comment by leofeyer** [comment]
Which extension do I have to install to reproduce this?

**Comment by agoat** [comment]
You can try it with one of mine: https://packagist.org/packages/agoat/contao-customcontentelements

**Comment by leofeyer** [comment]
You mean `agoat/contao-customcontentelements-bundle`? I figured I'd need something that does not end with `-bundle` to reproduce this?

**Comment by agoat** [comment]
My extensions (except here on Github) do not end with `-bundle`. 
Composer install packages with the name registered on packagist (respectively the name in composer.json), or am I wrong?

**Comment by leofeyer** [comment]
Fixed in c3eca067873434298babfb04ec2f602ca8230dd8.
